packages/core/src/configuration.ts

```diff
@@ -61,6 +61,10 @@ const legend: LegendOptions = {
  * Grid options
  */
 export const grid: GridOptions = {
+	// set enable to false will not draw grid and stroke of grid backdrop
+	xGridEnabled: true,
+	yGridEnabled: true,
+
```

Comment:
i think this should go within the x and y configurations liks this 
```
x: {
  enabled: true,
  numberOfTicks: 15
}
```

or maybe "visible" is a better word... since its still enabled/functional

Reply:
Based on @theiliad 's input. I will use "enabled".